test/item/uri.c

```diff
@@ -29,7 +29,7 @@ static
 void
 test_uri_new_free (void)
 {
-	g_autoptr(JURI) uri = NULL;
+	G_GNUC_UNUSED g_autoptr(JURI) uri = NULL;
```

Comment:
Why? It is used two lines later.

Reply:
The warning is write, but not read.
Found by a static code analyzer.